layouts/shortcodes/tab.html

```diff
@@ -1,8 +1,8 @@
 <!-- Make sure that we are enclosed within a tabpane shortcode block -->
 {{ with $.Parent }}
-{{- if ne $.Parent.Name "tabpane" -}}
-{{- errorf "tab must be used within a tabpane block" -}}
-{{- end -}}
+  {{- if ne $.Parent.Name "tabpane" -}}
+    {{- errorf "shortcode 'tab' must be used within a 'tabpane' block" -}}
+  {{- end -}}
 {{- end -}}
 
 <!-- Prefill header if not given as parameter -->
```

Comment:
I wrote this comment before but I don't see it, so I guess I forgot to click "submit"  `¯\_(ツ)_/¯`

It would be nice to be able to use an unnamed parameter to set the tab title. For example:

```
{{% tab "My Tab Title" %}}
Some stuff here
{{% /tab %}}
 ```

So, that means checking if there's an unnamed parameter and use it as title. 

This is what I have in mind (I was already working on some ideas before I saw your PR):

```
<!-- Prefill header if not given as parameter -->
{{ if and (not .IsNamedParams) (.Get 0) }}
  {{ .Scratch.Set "header" (.Get 0) }}
{{ else }}
  {{ .Scratch.Set "header" (default (printf "Tab %v" ( add $.Ordinal 1)) (.Get "header")) }}
{{ end }}
{{ $header := (.Scratch.Get "header")  }}
```

Reply:
Thanks for you suggestion, good idea. Implemented as requested,